server/controllers/authController.js

```diff
@@ -114,13 +114,52 @@ class AuthController {
               email,
               username,
               role,
+              lineManager,
               isVerified
             }
           }
         }));
       },
     )(req, res);
   }
+
+  /**
+     * This method handle the sign request.
+     * @param {object} req The user's request.
+     * @param {object} res The response.
+     * @returns {object} The status and some data of the user.
+     */
+  static async socialAuth(req, res) {
+    const {
+      id, firstname, lastname, email, role, isVerified, lineManager
+    } = req.user;
+
+    const token = await AuthHelper.createToken({
+      id,
+      firstname,
+      lastname,
+      lineManager,
+      email,
+      role,
+      isVerified
+    });
```

Comment:
The username should be included as well.

Reply:
Feedback Implemented!